pkg/webhook/webhook.go

```diff
@@ -42,6 +42,7 @@ var (
 		"0.4.0": []string{"0.4.0"},
 		"0.5.0": []string{"0.5.0", "0.6.0"},
 		"0.6.0": []string{"0.6.0"},
```

Comment:
Need to add an entry for 0.7.0 against the 0.6.0 and 0.5.0 keys so that we can support upgrade from 0.6.0 to 0.7.0 and from 0.5.0 to 0.7.0.
Please also test this works

Reply:
done